.ci/create-release-github.sh

```diff
@@ -7,4 +7,5 @@ OPERATOR_VERSION=$(git describe --tags)
 gh release create \
     -t "Release ${OPERATOR_VERSION}" \
     "${OPERATOR_VERSION}" \
-    'dist/splunk-otel-operator.yaml#Installation manifest for Kubernetes'
+    'dist/splunk-otel-operator.yaml#Installation manifest for Kubernetes' \
+    'dist/splunk-otel-operator-openshift.yaml#Installation manifest for Kubernetes for OpenShift'
```

Comment:
`     'dist/splunk-otel-operator-openshift.yaml#Installation manifest for OpenShift'`